js/ui/slider.d.ts

```diff
@@ -159,6 +160,12 @@ export interface dxSliderBaseOptions<TComponent> extends dxTrackBarOptions<TComp
        */
       showMode?: TooltipShowMode;
     };
+    /**
+     * @docid
+     * @default 'instant'
+     * @public
+     */
+     SliderCallValueChange: SliderCallValueChange;
```

Comment:
option name is not `SliderCallValueChange`, it's `callValueChange`.
`SliderCallValueChange` is just a type of this option value.